_commands/basics/wc.md

```diff
@@ -2,23 +2,74 @@
 ---
 
 wc
--------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+--
+
+`wc` prints newline, word, and byte counts for each FILE, and a total if more than one FILE is specified. With no FILE, or when FILE is a dash ("-"), wc operates on standard input. (A word is a non-zero-length sequence of characters delimited by white space.)
 
-<!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ wc [OPTION]... [FILE]...
+$ cd [OPTION]... --files0-from=F
```

Comment:
Not sure why `cd` is here? This can be removed for the example command. I think your first example down below could be a good sample command.

For example:

> ``` bash
> $ wc tecmint.txt
> 12  16 112 tecmint.txt   # number of lines, words, and bytes in the file
> ```